internal/build/docker.go

```diff
@@ -231,10 +227,20 @@ func (l *localDockerBuilder) buildFromDf(ctx context.Context, df Dockerfile, pat
 	}()
 	result, err := readDockerOutput(ctx, imageBuildResponse.Body)
 	if err != nil {
-		return "", fmt.Errorf("ImageBuild: %v", err)
+		return nil, fmt.Errorf("ImageBuild: %v", err)
+	}
+
+	digest, err := getDigestFromAux(*result)
+	if err != nil {
+		return nil, fmt.Errorf("getDigestFromAux: %v", err)
+	}
+
+	nt, err := l.TagDocker(ctx, ref, digest)
```

Comment:
Here's where we do the tag inside of the build.

Reply:
also why does this function now take the ref to the existing docker image??